src/cluster_undersample/interface_tables.jl

```diff
@@ -0,0 +1,76 @@
+### Cluster Undersampling TableTransforms Interface
+# interface struct
+struct ClusterUndersampler{T} <: Transform
+    y_ind::Integer
+    mode::AbstractString
+    ratios::T
+    maxiter::Integer
+    rng::Integer
+    try_perserve_type::Bool
+end
+
+"""
+Instantiate a naive ClusterUndersampler table transform
+
+# Arguments
+
+- `y_ind::Integer`: The index of the column containing the labels in the table
+
+- `mode::AbstractString="nearest`: If `center` then the undersampled data will consist of the centriods of 
+    each cluster found. Meanwhile, if `nearest` then it will consist of the nearest neighbor of each centroid.
```

Comment:
```suggestion
- `mode::AbstractString="nearest`: If `"center"` then the undersampled data will consist of the centriods of 
    each cluster found; if `"nearest"` then it will consist of the nearest neighbor of each centroid.
```